src/Authentication/docs/Add-MgEnvironment.md

```diff
@@ -0,0 +1,124 @@
+﻿---
+external help file: Microsoft.Graph.Authentication.dll-Help.xml
+Module Name: Microsoft.Graph.Authentication
+online version: https://learn.microsoft.com/en-us/powershell/module/microsoft.graph.authentication/add-mgenvironment
+schema: 2.0.0
+---
+
+# Add-MgEnvironment
+
+## SYNOPSIS
+{{ Fill in the Synopsis }}
```

Comment:
Incomplete synopsis, description, and examples for handwritten cmdlets. We shouldn't ship the feature with placeholders. @timayabi2020, please create an issue to add the missing descriptions to the help files. You can find these in the C# code and their associated PRs - https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/69f727bfff4e6d9fda7a7300d9954b1c9f3a8d5a/src/Authentication/Authentication/Cmdlets/Environment/AddMgEnvironment.cs#L17 and https://github.com/microsoftgraph/msgraph-sdk-powershell/pull/425